---
layout: post
title: Genomics of Microbes and Microbiomes Overview
category:
    - papers
tags:
    - youtube
    - papers
published: true
---

Listed below is a brief overview of the talk and the papers referenced
throughout the talk.

<!-- break -->

Recently, the NHGRI published a series on Current Topics in Genome Analysis.
One of the talks was of special interest to me, [Genomics of Microbes and
Microbiomes][yt] by [Julie Segre][segre].

[yt]: https://www.genome.gov/10000354/
[segre]: https://www.genome.gov/10000354/

## Topic 1 Bacterial Diversity Studies with 16S rRNA Sequencing

Important issues to consider in experiment:

- Study design?
- Sequencing platform?
- 16S rRNA region?
- Number of reads?
- Hidden technical issues?
- Analysis tools?
- Visualize data?
- Comparison to previous studies?
- What information will yield testable hypothesis?

Describing diversity:

- richeness - number of OTUs
- evenness - Shannon equitability index
- Shannon diversity index - accounts for both richness and evenness of OTUs

Papers:

- Peterson, Daniel A., et al. ["Metagenomic Approaches for Defining the
  Pathogenesis of Inflammatory Bowel Diseases"][peterson2008] *Cell Host &
  Microbe* 3.6 (2008): 417-427.
- Siqueira, José F., Ashraf F. Fouad, and Isabela N. Roˆcas. ["Pyrosequencing
  as a tool for better understanding of human microbiomes."][siqueira2012]
  *Journal of Oral Microbiology* 4 (2012).
- Kong, Heidi H. ["Skin microbiome: genomics-based insights into the diversity
  and role of skin microbes."][kong2011] *Trends in Molecular Medicine* 17.6
  (2011): 320-328.
- Castillo, Marisol, et al. ["Quantification of total bacteria, enterobacteria
  and lactobacilli populations in pig digesta by real-time PCR."][castillo2006]
  *Veterinary Microbiology* 114.1 (2006): 165-170.
- Kozich, James J., et al. ["Development of a dual-index sequencing strategy
  and curation pipeline for analyzing amplicon sequence data on the MiSeq
  Illumina sequencing platform."][kozich2013] *Applied and environmental
  Microbiology* 79.17 (2013): 5112-5120.
- Caporaso, J. Gregory, et al. ["Ultra-high-throughput microbial community
  analysis on the Illumina HiSeq and MiSeq platforms."][caporaso2012] *The ISME
  Journal* 6.8 (2012): 1621-1624.
- Fadrosh, Douglas W., et al. ["An improved dual-indexing approach for
  multiplexed 16S rRNA gene sequencing on the Illumina MiSeq
  platform."][fadrosh2014] *Microbiome* 2.1 (2014): 1.
- Conlan, Sean, Heidi H. Kong, and Julia A. Segre. ["Species-level analysis of
  DNA sequence data from the NIH Human Microbiome Project."][conlon2012] *PloS
  One* 7.10 (2012): e47075.
- Ravel, Jacques, et al. ["Vaginal microbiome of reproductive-age
  women."][ravel2011] *Proceedings of the National Academy of Sciences*
  108.Supplement 1 (2011): 4680-4687.
- Wang, Qiong, et al. ["Naive Bayesian classifier for rapid assignment of rRNA
  sequences into the new bacterial taxonomy."][wang2007] *Applied and
  Environmental Microbiology* 73.16 (2007): 5261-5267.
- Quast, Christian, et al. ["The SILVA ribosomal RNA gene database project:
  improved data processing and web-based tools."][quast2012] *Nucleic Acids
  Research* 41.D1 (2013): D590-D596.
- Ludwig, Wolfgang, et al. ["ARB: a software environment for sequence
  data."][ludwig2004] *Nucleic Acids Research* 32.4 (2004): 1363-1371.
- Haas, Brian J., et al. ["Chimeric 16S rRNA sequence formation and detection
  in Sanger and 454-pyrosequenced PCR amplicons."][haas2011] *Genome Research*
  21.3 (2011): 494-504.
- Jumpstart Consortium Human Microbiome Project Data Generation Working Group.
  ["Evaluation of 16S rDNA-based community profiling for human microbiome
  research."][jumpstart2012] *PloS One* 7.6 (2012): e39315.
- Grice, Elizabeth A., and Julia A. Segre. ["The human microbiome: our second
  genome."][grice2012] *Annual Review of Genomics and Human Genetics* 13
  (2012): 151.
- Oh, Julia, et al. ["Shifts in human skin and nares microbiota of healthy
  children and adults."][oh2012] *Genome Medicine* 4.10 (2012): 1.
- Ley, Ruth E., et al. ["Obesity alters gut microbial ecology."][ley2005]
  *Proceedings of the National Academy of Sciences of the United States of
  America* 102.31 (2005): 11070-11075.
- Scharschmidt, Tiffany C., et al. ["Matriptase-deficient mice exhibit
  ichthyotic skin with a selective shift in skin
  microbiota."][scharschmidt2009] *Journal of Investigative Dermatology* 129.10
  (2009): 2435-2442.
- Hamady, Micah, and Rob Knight. ["Microbial community profiling for human
  microbiome projects: Tools, techniques, and challenges."][hamady2009] *Genome
  Research* 19.7 (2009): 1141-1152.
- Kuczynski, Justin, et al. ["Experimental and analytical tools for studying
  the human microbiome."][kuczynski2012] *Nature Reviews Genetics* 13.1 (2012):
  47-58.

## Topic 2 Fungal Diversity

- Grice, Elizabeth A., et al. ["Topographical and temporal diversity of the
  human skin microbiome."][grice2009] *Science* 324.5931 (2009): 1190-1192.
- Findley, Keisha, et al. ["Topographic diversity of fungal and bacterial
  communities in human skin."][findley2013] *Nature* 498.7454 (2013): 367-370.

## Topic 3 Bacterial Genomes

Need to consider similar issues as 16S rRNA, but also the following:

- Existing reference genome?
- Sequencing depth?

Papers:

- Hunt, Martin, et al. ["A comprehensive evaluation of assembly scaffolding
  tools."][hunt2014] *Genome Biology* 15.3 (2014): 1.
- Tettelin, Hervé, et al. ["Genome analysis of multiple pathogenic isolates of
  Streptococcus agalactiae: implications for the microbial
  'pan-genome'."][tettelin2005] *Proceedings of the National Academy of
  Sciences of the United States of America* 102.39 (2005): 13950-13955.
- Krzywinski, Martin, et al. ["Circos: an information aesthetic for comparative
  genomics."][krzywinski2009] *Genome Research* 19.9 (2009): 1639-1645.

## Topic 4 Metagenomics

Databases:

- KEGG
- MetaCyc
- the Gene Ontology
- eggNOG

Goals:

- Who's there and abundance?
- Function?
    - Genes
    - Pathways
    - Strains
- Recover genomes?
- Novel pathogenic organisms?

Papers:

- Hugenholtz, Philip, and Gene W. Tyson. ["Microbiology:
  metagenomics."][hugenholtz2008] *Nature* 455.7212 (2008): 481-483.
- Warnecke, Falk, et al. ["Metagenomic and functional analysis of hindgut
  microbiota of a wood-feeding higher termite."][warnecke2007] *Nature* 450.7169
  (2007): 560-565.
- Hess, Matthias, et al. ["Metagenomic discovery of biomass-degrading genes and
  genomes from cow rumen."][hess2011] *Science* 331.6016 (2011): 463-467.
- Segata, Nicola, et al. ["Metagenomic microbial community profiling using
  unique clade-specific marker genes."][segata2012] *Nature Methods* 9.8
  (2012): 811-814.
- Sunagawa, Shinichi, et al. ["Metagenomic species profiling using universal
  phylogenetic marker genes."][sunagawa2013] *Nature Methods* 10.12 (2013):
  1196-1199.
- Abubucker, Sahar, et al. ["Metabolic reconstruction for metagenomic data and
  its application to the human microbiome."][abubucker2012] *PLoS Comput Biol*
  8.6 (2012): e1002358.
- Human Microbiome Project Consortium. ["Structure, function and diversity of
  the healthy human microbiome."][hmp2012] *Nature* 486.7402 (2012): 207-214.
- Ghai, Rohit, et al. ["New abundant microbial groups in aquatic hypersaline
  environments."][ghai2011] *Scientific Reports* 1 (2011).
- Karlsson, Fredrik H., et al. ["Gut metagenome in European women with normal,
  impaired and diabetic glucose control."][karlsson2013] *Nature* 498.7452
  (2013): 99-103.
- Oh, Julia, et al. ["Biogeography and individuality shape function in the
  human skin metagenome."][oh2014] *Nature* 514.7520 (2014): 59-64.
- Zhu, Ana, et al. ["Inter-individual differences in the gene content of human
  gut bacterial species."][zhu2015] *Genome Biology* 16.1 (2015): 1.
- Naccache, Samia N., et al. ["A cloud-compatible bioinformatics pipeline for
  ultrarapid pathogen identification from next-generation sequencing of
  clinical samples."][naccache2014] *Genome Research* 24.7 (2014): 1180-1192.
- Wilson, Michael R., et al. ["Actionable diagnosis of neuroleptospirosis by
  next-generation sequencing."][wilson2014] *New England Journal of Medicine*
  370.25 (2014): 2408-2417.

## Topic 4 Sequencing Technology

Illumina MiSeq dominates, but PacBio gives long reads for reference genomes.
There may be talk of MinION from Oxford Nanopore in teh future for long read
sequencing.

- Byrd, Allyson L., and Julia A. Segre. ["Adapting Koch's
  postulates."][byrd2016] *Science* 351.6270 (2016): 224-226.

[peterson2008]: http://dx.doi.org/10.1016/j.chom.2008.05.001
[siqueira2012]: http://dx.doi.org/10.3402/jom.v4i0.10743
[kong2011]: http://dx.doi.org/10.1016/j.molmed.2011.01.013
[castillo2006]: http://dx.doi.org/10.1016/j.vetmic.2005.11.055
[kozich2013]: http://dx.doi.org/10.1128/AEM.01043-13
[caporaso2012]: http://dx.doi.org/10.1038/ismej.2012.8
[fadrosh2014]: http://dx.doi.org/10.1186/2049-2618-2-6
[conlon2012]: http://dx.doi.org/10.1371/journal.pone.0047075
[ravel2011]: http://dx.doi.org/10.1073/pnas.1002611107
[wang2007]: http://dx.doi.org/10.1128/AEM.00062-07
[quast2012]: http://dx.doi.org/10.1093/nar/gks1219
[ludwig2004]: http://dx.doi.org/10.1093/nar/gkh293
[haas2011]: http://dx.doi.org/10.1101/gr.112730.110
[jumpstart2012]: http://dx.doi.org/10.1371/journal.pone.0039315
[grice2012]: https://dx.doi.org/10.1146/annurev-genom-090711-163814
[oh2012]: http://dx.doi.org/10.1186/gm378
[ley2005]: http://dx.doi.org/10.1073/pnas.0504978102
[scharschmidt2009]: http://dx.doi.org/10.1038/jid.2009.104
[hamady2009]: http://dx.doi.org/10.1101/gr.085464.108
[kuczynski2012]: http://dx.doi.org/10.1038/nrg3129
[grice2009]: https://doi.org/10.1126/science.1171700
[findley2013]: http://dx.doi.org/10.1038/nature12171
[hunt2014]: http://dx.doi.org/10.1186/gb-2014-15-3-r42
[tettelin2005]: http://dx.doi.org/10.1073/pnas.0506758102
[krzywinski2009]: http://dx.doi.org/10.1101/gr.092759.109
[hugenholtz2008]: http://dx.doi.org/10.1038/455481a
[warnecke2007]: http://dx.doi.org/10.1038/nature06269
[hess2011]: http://dx.doi.org/10.1126/science.1200387
[segata2012]: http://dx.doi.org/10.1038/nmeth.2066
[sunagawa2013]: http://dx.doi.org/10.1038/nmeth.2693
[abubucker2012]: http://dx.doi.org/10.1371/journal.pcbi.1002358
[hmp2012]: http://dx.doi.org/10.1038/nature11234
[ghai2011]: http://dx.doi.org/10.1038/srep00135
[karlsson2013]: http://dx.doi.org/10.1038/nature12198
[oh2014]: http://dx.doi.org/10.1038/nature13786
[zhu2015]: http://dx.doi.org/10.1186/s13059-015-0646-9
[naccache2014]: http://dx.doi.org/10.1101/gr.171934.113
[wilson2014]: http://dx.doi.org/10.1056/NEJMoa1401268
[byrd2016]: http://dx.doi.org/10.1126/science.aad6753
