---
layout: post
title: Twitter notes of #SACMDA3
categories: [conferences, microbiome]
tags: [twitter, conferences, microbiome, statistics]
---

Several weeks ago,
the [3rd Workshop on Statistical and Algorithmic Challenges in Microbiome Data Analysis](//www.simonsfoundation.org/event/3rd-workshop-on-statistical-and-algorithmic-challenges-in-microbiome-data-analysis/)
was hosted at the
[Simons Foundation](//www.simonsfoundation.org/).
I did not attend,
but with the magic of Twitter and the internet,
I could follow along with the generous efforts of live-tweeting with the
hashtag
[`#SACMDA3`](https://twitter.com/search?f=tweets&vertical=default&q=%23sacmda3&src=typd).

Here are my notes on what happened.

## First day

A simple but crucial question in the world of microbiome science.

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Incredibly impressed by the morning talks at <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a><a href="https://twitter.com/hashtag/SACMDA?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA</a>. ‚ÄúWhat‚Äôs in my sample?‚Äù is still an open and important question ‚Äî and it‚Äôs to hear from both new and senior researchers working on strain id &amp; tax. assignment qtns! Great work, Nidhi, Chris, Rachel &amp; Francesco!!</p>&mdash; Amy Willis (@AmyDWillis) <a href="https://twitter.com/AmyDWillis/status/1112731466620944384?ref_src=twsrc%5Etfw">April 1, 2019</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

Great thread, summarizing the first day of the workshop, along with links to
papers, GitHub and Bitbucket code repositories, and comments.

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Excited to be at <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a>, third annual-ish Workshop on Statistical and Algorithmic Challenges in Microbiome Data Analysis, sponsored by <a href="https://twitter.com/MITMicrobiome?ref_src=twsrc%5Etfw">@MITMicrobiome</a> and <a href="https://twitter.com/SimonsFdn?ref_src=twsrc%5Etfw">@SimonsFdn</a> <a href="https://twitter.com/FlatironCCB?ref_src=twsrc%5Etfw">@FlatironCCB</a> <a href="https://t.co/Gy4owbvZtK">https://t.co/Gy4owbvZtK</a></p>&mdash; Claire Duvallet (@cduvallet) <a href="https://twitter.com/cduvallet/status/1112734439824654341?ref_src=twsrc%5Etfw">April 1, 2019</a></blockquote>

A reminder of our role as computationalists.

<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr">Another lol moment (for me) here at <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a>: &quot;Goal: narrow down the search space for experimentalists.&quot; <a href="https://twitter.com/hashtag/BenevolentComputationalists?src=hash&amp;ref_src=twsrc%5Etfw">#BenevolentComputationalists</a></p>&mdash; Claire Duvallet (@cduvallet) <a href="https://twitter.com/cduvallet/status/1112835864097419267?ref_src=twsrc%5Etfw">April 1, 2019</a></blockquote>

> "**A big challenge in computational metagenomics**: Lacking consensus about
> benchmarking datasets, evaluation procedures, and metrics complicates prior
> performance assessments."

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Echo that! <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a> <a href="https://t.co/7vHXUmeunc">pic.twitter.com/7vHXUmeunc</a></p>&mdash; Erika Ganda (@EKGanda) <a href="https://twitter.com/EKGanda/status/1112839528845381632?ref_src=twsrc%5Etfw">April 1, 2019</a></blockquote>

## Second day

Another great thread by Clarie Duvallet for the second day.

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Day 2 of <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a> kicked off by <a href="https://twitter.com/KnightLabNews?ref_src=twsrc%5Etfw">@KnightLabNews</a> emphasizing the need to get message about appropriate statistical techniques for data analysis to the entire <a href="https://twitter.com/hashtag/microbiome?src=hash&amp;ref_src=twsrc%5Etfw">#microbiome</a> community. Ends with: anyone here wanna write a review paper with him, <a href="https://twitter.com/ejalm?ref_src=twsrc%5Etfw">@ejalm</a>, and <a href="https://twitter.com/RichBonneauNYU?ref_src=twsrc%5Etfw">@RichBonneauNYU</a>? üòâ</p>&mdash; Claire Duvallet (@cduvallet) <a href="https://twitter.com/cduvallet/status/1113067777928192000?ref_src=twsrc%5Etfw">April 2, 2019</a></blockquote>

This blog post on
[Statistical Methods for Microbiome Data](https://pratheepaj.github.io/research/statistical-methods-for-microbiome/)
was mentioned,
especially on the section for a novel method for contamination removal.

> "Every metagenomic measurement is **wrong**."

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">This public service announcement brought to you by <a href="https://twitter.com/bejcal?ref_src=twsrc%5Etfw">@bejcal</a> and <a href="https://twitter.com/mikemc423?ref_src=twsrc%5Etfw">@mikemc423</a> <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a> <a href="https://t.co/Sw1nNyuKIk">pic.twitter.com/Sw1nNyuKIk</a></p>&mdash; Amy Willis (@AmyDWillis) <a href="https://twitter.com/AmyDWillis/status/1113084303519502344?ref_src=twsrc%5Etfw">April 2, 2019</a></blockquote>

> Challenges when working with Longitudinal Multidomain data
>
> - Data Quality: Heterogeneity, unwanted sourcecs of variation.
> - Building modles from the data.
> - Interpretation of analytic output.
> - Multiple (dual) dependencies
> - Multidomain, need for registration.
> - Uncertainty quantification and inference.
> - Reproducibility of results across labs, experimental conditions and users.

<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Susan Holmes talking about the many challenges when working with data. <a href="https://twitter.com/hashtag/SACMDA3?src=hash&amp;ref_src=twsrc%5Etfw">#SACMDA3</a> <a href="https://t.co/KaWcw9gDOV">pic.twitter.com/KaWcw9gDOV</a></p>&mdash; Erika Ganda (@EKGanda) <a href="https://twitter.com/EKGanda/status/1113146419295207425?ref_src=twsrc%5Etfw">April 2, 2019</a></blockquote>


## Software packages mentioned

- CONCOCT
    - "Unsupervised binning of metagenomic contigs by using nucleotide
      composition, coverage data in multiple samples and linkage data from
      paired end reads."
    - [Paper](//www.ncbi.nlm.nih.gov/pubmed/25218180)
    - [GitHub](//github.com/BinPro/CONCOCT)
- Pandora
    - "Pan-genome inference with long error-prone sequencing reads"
    - In active development
    - [GitHub](//github.com/rmcolq/pandora)
- DESMAN
    - "DESMAN identifies variants in core genes and uses co-occurrence across
      samples to link variants into haplotypes and abundance profiles."
    - This tool was not mentioned directly, but appears to be the predecessor
      to
      [a new tool that was mentioned](https://twitter.com/RichBonneauNYU/status/1112711501050200064)
    - [Paper](//doi.org/10.1186/s13059-017-1309-9)
- PhyloPhlAn (Version 2)
    - "PhyloPhlAn is a computational pipeline for reconstructing highly
      accurate and resolved phylogenetic trees based on whole-genome sequence
      information."
    - [Paper](//doi.org/10.1038/ncomms3304)
    - [Bitbucket](https://bitbucket.org/nsegata/phylophlan/wiki/Home)
- FEAST
    - "Fast expectation maximization source tracking"
    - [GitHub](https://github.com/cozygene/feast)
- rhapsody
    - "Neural networks for microbe-metabolite interaction analysis"
    - [GitHub](https://github.com/biocore/rhapsody)
- mixOmics
    - "mixOmics: An R package for ‚Äòomics feature selection and multiple data
      integration"
    - [Website](http://mixomics.org/)
    - [Paper](https://doi.org/10.1371/journal.pcbi.1005752)

## Papers

- [Pasolli et al., "Extensive Unexplored Human Microbiome Diversity Revealed by Over 150,000 Genomes from Metagenomes Spanning Age, Geography, and Lifestyle". Cell (2019)](https://doi.org/10.1016/j.cell.2019.01.001)
- [Wirbel et al., "Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer". Nature Medicine (2019)](https://doi.org/10.1038/s41591-019-0406-6)
- [Sczyrba et al.,"Critical Assessment of Metagenome Interpretation‚Äîa benchmark of metagenomics software". Nature Methods (2017)](https://doi.org/10.1038/nmeth.4458)
- [McLaren, Willis, Callahan. "Consistent and correctable bias in metagenomic sequencing measurements" bioRxiv (2019)](https://doi.org/10.1101/559831)
- [Fernandes et al., "A reproducible effect size is more useful than an irreproducible hypothesis test to analyze high throughput sequencing datasets" arXiv (2018)](https://arxiv.org/abs/1809.02623)
- [Washburne et al., "Phylogenetic factorization of compositional data yields lineage-level associations in microbiome datasets". PeerJ (2017)](https://peerj.com/articles/2969/)
- [Washburne et al., "Phylofactorization: a graph-partitioning algorithm to identify phylogenetic scales of ecological data". bioRxiv](https://doi.org/10.1101/235341)
- [Rivera-Pinto et al., "Balances: a New Perspective for Microbiome Analysis"](https://doi.org/10.1128/mSystems.00053-18)
- [Silverman et al., "Bayesian Multinomial Logistic Normal Models through Marginally Latent Matrix-T Processes". arXiv (2019)](https://arxiv.org/abs/1903.11695)

## Reflection

There was a lot of concepts, papers, and code repositories I couldn't cover.
Feel free to browse
[the Twitter feed](https://twitter.com/search?f=tweets&vertical=default&q=%23sacmda3&src=typd)
for more.

## Things to share

### Sorting lines in Vim

You can
[sort lines in Vim](https://vim.fandom.com/wiki/Sort_lines)
with `:{range}sort u`.
The `u` is to remove duplicate lines.
More can be found at `:help sort`.

### Spelling files created by Vim

For UTF-8 files, Vim creates two files:

- `en.utf-8.add`
- `en.utf-8.add.spl`

One discussion about them
[suggest one is plain text and the other is the corresponding compiled binary](https://www.reddit.com/r/vim/comments/5zhpre/).
I was editing my spelling files and was curiou about the difference between
them.

The `en.utf-8.add` file is plain text and I have version controlled it for
consistence across my computers.

The compiled `.spl` file can be recreated using

```vim
:mkspell ~/.vim/spell/en.uft-8.add
```
